### PR TITLE
Adding to the exclusion service and added regex to discount emojis an…

### PR DESCRIPTION
**Pull request by mairlot**
…d users from the message length filter

### CODE REVIEW

File: src/server/kudos/messageExclusionService.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const botFilter = message => !message.bot_id;
+const includeRegex = message => message.text.match(/(thank|kudo|cheers|congratulat|hats off)/i);
+const excludeRegex = /(be open to receiving|It would be great if one|No kudos necessary|Just to clarify|for starting this channel|pretty darn impressive|KudosBot has also decided|Currently KudosBot sends the Kudos summary|kudos_bot|Kudos bot is becoming|sassiness module|creating a poll but I have two questions|set the channel description|set the channel topic|Since last Friday we've had 14 kudos messages|How does it work|just start using the channel|in tandem with the existing feedback)/i;
+const messageFilter = message => !message.text.match(excludeRegex);
+const excludeEmojisAndUsersRegex = /(:[^:]*:)|(<[^<]*>)/g;
```
Comment (tonyhkly) [inline src/server/kudos/messageExclusionService.js:5]: Nice regex 👌

File: src/server/kudos/messageExclusionService.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const botFilter = message => !message.bot_id;
+const includeRegex = message => message.text.match(/(thank|kudo|cheers|congratulat|hats off)/i);
+const excludeRegex = /(be open to receiving|It would be great if one|No kudos necessary|Just to clarify|for starting this channel|pretty darn impressive|KudosBot has also decided|Currently KudosBot sends the Kudos summary|kudos_bot|Kudos bot is becoming|sassiness module|creating a poll but I have two questions|set the channel description|set the channel topic|Since last Friday we've had 14 kudos messages|How does it work|just start using the channel|in tandem with the existing feedback)/i;
```
Comment (tonyhkly) [inline src/server/kudos/messageExclusionService.js:3]: Wish there was a nicer solution to this text exclusion

Comment (mairlot) [reply src/server/kudos/messageExclusionService.js:3]: Give it time and there will be...

**Comment by tonyhkly** [comment]
I was in the middle of a meeting and forgot to approve soz

**Comment by mairlot** [comment]
No worries, still getting used to Github so thought I might just be missing where the approval was but knew you'd looked at it, so all good

**Pull request merged**